_posts/2015-10-15-best-practices-for-distributed-teams.md

```diff
@@ -0,0 +1,208 @@
+forward.”
+
+![A grid of 18F team members using video chat to attend a meeting.](/assets/blog/distributed-teams/18f-team-acano.jpg)
+*Part of the 18F team at the weekly all-hands meeting.*
+
+We share our screens frequently.
+--------------------------------
+
+Have a question for a coworker? Two minutes later, we might be sharing a
+screen together, using programs such as appear.in, Google Hangouts, Screenhero, and
+plain-old screencasting. This allows us to ask for feedback, show off
+designs, and [pair
+program](https://18f.gsa.gov/2015/05/04/pair-programming-why-two-heads-are-better-than-one/)
+with each other.
+
+We have face-to-face meetings at least once a week.
+---------------------------------------------------
+
+All of our teams meet with each other face-to-face on a weekly basis, if
+not more frequently. We use high-quality microphones so that distributed
+teams can hear things clearly. Video allows us to communicate more
+expressively, and to give non-verbal cues like a thumbs up signal. It
+fosters stronger connections and helps us to be responsive to each
+other.
+
+But sometimes things don’t work well. Audio drops out. Video fails.
+
+As Jacob Harris, a developer who works in D.C., puts it: “I think
+everybody here agrees that the most implausible science-fiction aspect
+of Star Wars is how smoothly all the Jedi videoconferencing works across
+the galaxy.”
+
+So we also make contingency plans in case video or audio drops out — and
+even when it doesn’t. For example, in our weekly all-hands meeting,
+which everyone at 18F attends, we have video with sound, a phone option,
+and we transcribe everything in real time in a Slack room, so that
+remote participants (or people away from their desks) can keep up.
+
+Jacob’s team uses several measures to compensate when videochat doesn’t
+come through clearly.
+
+“A shared Google document with an agenda helps people to know what the
+meeting will be about in advance,” he says. “And anything that is
+discussed / decided in the meeting must be added to that document.
+Similarly, we request that any bugs or other coding work must be
+documented using GitHub's mechanisms for issues, pull requests. This
+makes it easier to remember what we're working on and keeps our actions
+public in case someone wasn't able to hear what we were talking about.”
+
+We make our work transparent to each other.
+-------------------------------------------
+
+We’re committed to making all our work, discussions, and decisions
+available to everyone in the organization at all times. We do this by
+having:
+
+-   **Real-time chat (aka, virtual water cooler)***.* 18F uses [Slack](https://slack.com/) to “hang out” virtually. This area is open all day for people to drop in and out as their schedule allows. The important thing is that everyone is invested in using it consistently, something that managers and team leads may need to continually reinforce. Everyone uses chat, even people in the office.
```

Comment:
Looks like we're bolding the text and italicizing the punctuation? I'm guessing this is a Google Docs/pandoc conversion problem?